packages/api/src/car.js

```diff
@@ -104,6 +104,9 @@ export async function carPost (request, env, ctx) {
   }
 
   const blob = await request.blob()
+  const bytes = new Uint8Array(await blob.arrayBuffer())
+  const reader = await CarReader.fromBytes(bytes)
+  const chunkSize = await getBlocksSize(reader)
```

Comment:
We could use a CarBlockIterator here and wrap request.body as a AsyncIterator to only unpack a block into mem at once?